### PR TITLE
NEW: CPButtonAnimator

### PR DESCRIPTION
**Pull request by cacaodev**
Animation of the following CPButton properties:
- textColor
- fontSize
- font (not supported by all browsers)

### CODE REVIEW

File: AppKit/CoreAnimation/CPViewAnimator.j [hunk 1]
```diff
@@ -219,4 +220,74 @@ var DEFAULT_CSS_PROPERTIES = nil;
...
+- (float)fontSize
+{
+    return [[self font] size];
+}
+
+@end
+
+@implementation CPFont (CPViewAnimator)
+
+- (CPFont)fontOfSize:(CPInteger)aSize
```
Comment (Dogild) [inline AppKit/CoreAnimation/CPViewAnimator.j]: This method should be "private", with `_`

File: AppKit/CoreAnimation/CPViewAnimator.j [hunk 2]
```diff
@@ -219,4 +220,74 @@ var DEFAULT_CSS_PROPERTIES = nil;
...
+        [_target setFont:font];
+        [[CPRunLoop currentRunLoop] performSelectors];
+    }];
+}
+
+@end
+
+@implementation _CPImageAndTextView (CPAnimatablePropertyContainer)
+
++ (CPArray)cssPropertiesForKeyPath:(CPString)aKeyPath
```
Comment (Dogild) [inline AppKit/CoreAnimation/CPViewAnimator.j]: This method should be "private", with _

**Comment by mrcarlberg** [comment]
@cacaodev Any progress on this?

**Comment by cacaodev** [comment]
I will close this because font and fontSize support is not fully supported in webkit.

**Comment by mrcarlberg** [comment]
Ok!

**Pull request closed without merging**